src/components/elections-graph/elections-graph.stories.js

```diff
@@ -0,0 +1,8 @@
+import "./elections-graph.component.js";
+
+export default {
+  title: "Components/Elections Graph",
```

Comment:
Can this be moved into an Elections sub folder for storybook
```suggestion
  title: "Components/Elections/Elections Graph",
```